src/main/java/EchoBot.java

```diff
@@ -0,0 +1,92 @@
+import java.util.ArrayList;
```

Comment:
Perhaps you want to move your EchoBot class under the duke package so that all classes are minimally under the duke package 😄